src/System.ValueTuple/tests/System.ValueTuple.Tests.builds

```diff
@@ -2,8 +2,15 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.ValueTuple.Tests.csproj" />
     <Project Include="System.ValueTuple.Tests.csproj">
+      <TestTFMs>netcoreapp1.1</TestTFMs>
+    </Project>
+    <Project Include="System.ValueTuple.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcore50;net46</TestTFMs>
+    </Project>
```

Comment:
I'm changing TestTFMs here to "netcoreapp1.0". I see that is how some other projects structure their tests (I'm looking at Microsoft.Win32.Primitives) and it makes some sense.